jwst/ami/utils.py

```diff
@@ -1296,39 +922,39 @@ def handle_bandpass(bandpass, throughput_model):
     Parameters
     ----------
     bandpass : Synphot spectrum or array, or None
```

Comment:
Fixed a bug in this function that caused output to have the wrong units if the input was a synphot spectrum